### PR TITLE
Document all API endpoints

### PR DESCRIPTION
**Pull request by interro**

### CODE REVIEW

File: tequilapi/endpoints/identities.go [hunk 1]
```diff
@@ -87,17 +103,17 @@ func (endpoint *identitiesAPI) List(resp http.ResponseWriter, request *http.Requ
...
 // swagger:operation POST /identities Identity createIdentity
 // ---
-// summary: Create new identity
+// summary: Creates new identity
 // description: Creates identity and stores in keystore encrypted with passphrase
 // parameters:
 //   - in: body
 //     name: body
-//     description: Parameters for creating new identity
+//     description: Parameter in body (passphrase) required for creating new identity
```
Comment (donce) [inline tequilapi/endpoints/identities.go]: Do we need to specify "in body"? This is transport, we should not couple description to transport.

Comment (interro) [reply tequilapi/endpoints/identities.go]: In swagger template this message looks more informative

File: tequilapi/endpoints/connection.go [hunk 2]
```diff
@@ -75,19 +95,33 @@ func NewConnectionEndpoint(manager connection.Manager, ipResolver ip.Resolver, s
...
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/errorMessage"
 func (ce *ConnectionEndpoint) Status(resp http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
 	statusResponse := toStatusResponse(ce.manager.Status())
 	utils.WriteAsJSON(statusResponse, resp)
 }
 
+// Creates starts new connection
```
Comment (zolia) [inline tequilapi/endpoints/connection.go]: should be "Create", shouldn't it be caught by linter? Did you run it?

File: tequilapi/endpoints/health_check.go [hunk 3]
```diff
@@ -25,16 +25,27 @@ import (
...
+
+	// example: 0.0.6
 	Version   string    `json:"version"`
 	BuildInfo buildInfo `json:"buildInfo"`
 }
 
 type buildInfo struct {
-	Commit      string `json:"commit"`
-	Branch      string `json:"branch"`
+	// example: <unknown>
```
Comment (zolia) [inline tequilapi/endpoints/health_check.go]: Why this struct doesn't have "// swagger:model"  heading?

File: tequilapi/endpoints/connection.go (hunk 2 above)
Comment (interro) [reply tequilapi/endpoints/connection.go]: fixed, linter configured to have some tolerance

File: tequilapi/endpoints/health_check.go (hunk 3 above)
Comment (interro) [reply tequilapi/endpoints/health_check.go]: added missing meta tags

File: bin/check_lint [hunk 4]
```diff
@@ -41,7 +41,7 @@ MESSAGES_ERROR=`golint --set_exit_status --min_confidence=1 ${ARGUMENTS} 2>/dev/
 MESSAGES_RECONFIGURE=()
 
 check_uncleaned_package "github.com/mysterium/node/identity" 9
-check_uncleaned_package "github.com/mysterium/node/tequilapi" 12
+check_uncleaned_package "github.com/mysterium/node/tequilapi" 11
```
Comment (Waldz) [inline bin/check_lint]: +1

File: tequilapi/endpoints/connection.go [hunk 5]
```diff
@@ -56,6 +56,26 @@ type statusResponse struct {
 	SessionID string `json:"sessionId,omitempty"`
 }
 
+// swagger:model
+type ipResponse struct {
```
Comment (Waldz) [inline tequilapi/endpoints/connection.go]: Very strange that structures are private.
Lets make them public, because it is so.

**Pull request merged**